draft-perrig-scion-overview.md

```diff
@@ -49,79 +49,81 @@ informative:
+ RPKI and BGPsec try to address Internet's above-mentioned security shortcomings, see also {{RFC6480}} and {{RFC8205}}. However, RPKI and BGPsec introduce additional challenges, as shortly described below.
 
-- **RPKI and Route Origin Authorizations**  
+- **RPKI and Route Origin Authorizations**
 Unfortunately, the Route Origin Authorizations (ROAs) provided by RPKI only prevent the simplest form of BGP hijacks, see [Attacks](#attack).
-- **Problems with BGPsec in partial deployment**  
-BGPsec only provides full security when all ASes consistently use and enforce it. In the current situation, where BGP is only partially deployed, it is not very effective. It can even cause instabilities and is prone to downgrade attacks, see {{LYCHEV2013}}.
-- **Problems with BGPsec in full deployment**  
-Also full deployment of BGPsec raises issues, such as the creation of wormholes and forwarding loops by attackers, or the introduction of circular dependencies, see {{LI2014}} and {{COOPER2013}}. RPKI and BGPsec together also cause issues for network sovereignty {{ROTHENBERGER2017}}. Additionally, BGPsec further exacerbates BGP’s scalability issues. Furthermore, prefix aggregation no longer works in BGPsec because the digital signatures are not aggregated.
+- **Problems with BGPsec in partial deployment**
+BGPsec only provides full security when all ASes consistently use and enforce it. In the current situation, where BGP is only partially deployed, it has a limited effectiveness. It can even cause instabilities and is prone to downgrade attacks, see {{LYCHEV2013}}.
+- **Problems with BGPsec in full deployment**
+Also full deployment of BGPsec raises issues, such as the creation of wormholes and forwarding loops by attackers, or the introduction of circular dependencies, see {{LI2014}} and {{COOPER2013}}. RPKI and BGPsec together also cause issues for network sovereignty {{ROTHENBERGER2017}}. Additionally, BGPsec further exacerbates BGP’s scalability issues (i.e., due to the additional overhead, and due to lack of prefix aggregation).
 
 ### Other Internet Issues
 
 #### Lack of Authentication
 
-Authenticating digital data is becoming increasingly important, as adversaries exploit the absence of authentication to inject malicious information. However, implementation of authentication is not strong in today's Internet:
+Authenticating digital data is becoming increasingly prevalent, as adversaries exploit the absence of authentication to inject malicious information. However, many network protocols lack authentication features:
 
-- Internet does not support sharing a secret key between two end hosts for secure end-to-end communication.  
-- Infrastructures added to provide authentication, such as RPKI/BGPsec, TLS {{RFC8446}}, and DNSSEC {{RFC4033}}, are all sensitive to the compromise of a single entity.  
-- The Internet Control Message Protocol (ICMP) does not even have an authenticated counterpart, see {{RFC4443}} and {{RFC0791}}.
```

Comment:
I hope this justifies why this is relevant.. ? Maybe we should mention some more ICMP attacks?

I'm not very familiar with them, I'm thinking about spoofed redirects, or destination unreachable, which could potentially have the impact of blocking certain kind of traffic. Maybe there is something else?